test/correctness/halide_buffer.cpp

```diff
@@ -75,7 +77,60 @@ void test_copy(Buffer<float> a, Buffer<float> b) {
     check_equal(a_window, b_window);
 }
 
+// // Fake "buffer" class that is shaped enough like Halide::Runtime::Buffer
```

Comment:
Did you intend to come back to this commented out code?

Reply:
Oops. Nope. Scalpel left in patient.